roles/pulp_database_config/tasks/main.yml

```diff
@@ -22,6 +26,29 @@
 - name: Flush handlers
   meta: flush_handlers
 
+- name: Create cert directory to hold token authentication key
+  file:
+    path: "{{ pulp_certs_dir }}"
+    state: directory
+    owner: "{{ pulp_user }}"
+    group: "{{ pulp_group }}"
+    mode: 0700
+  become: true
+
+- name: Import Pulp Database Encryption Key
+  copy:
+    src: "{{ pulp_db_fields_key }}"
+    dest: "{{ __pulp_db_fields_key_path }}"
+    owner: 'root'
+    group: "{{ pulp_group }}"
+    mode: 0640
```

Comment:
You removed the remote option, pulp_db_fields_key_remote.

If you think that is for the best, let's do that. But we need a removal notice.